infra/pipelines/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM prefecthq/prefect
+
+RUN pip install supervisor arrow graphistry pyarrow simplejson twarc neo4j
+RUN prefect agent install local > supervisord.conf
```

Comment:
If this is the executor, would it make more sense to layer off the pydata dependencies (rapids, sckit, ...)? We've been maintaining https://hub.docker.com/r/graphistry/graphistry-blazing for most